test/test_runtime.py

```diff
@@ -893,3 +893,16 @@ def test_get_galaxy_role_name_invalid() -> None:
         "role_name": False,  # <-- invalid data, should be string
     }
     assert _get_galaxy_role_name(galaxy_infos) == ""
+
+
+def test_runtime_has_playbook() -> None:
+    """Tests has_playbook method."""
+    runtime = Runtime(require_module=True)
+
+    assert not runtime.has_playbook("this-does-not-exit.yml")
```

Comment:
Are these supposed to be `this-does-not-exist.yml` or are there some kind of halting problem shenanigans going on?

Reply:
Just a random name that I know not to exist on disk.